core/scripts/cli/voting/batchCommitVotes.js

```diff
@@ -0,0 +1,38 @@
+const style = require("../textStyle");
+const { BATCH_MAX_COMMITS } = require("../../../../common/Constants");
+
+module.exports = async (newCommitments, votingContract, account) => {
```

Comment:
Note: this comment may be worth a TODO rather than you doing it in this PR (up to you).

It seems like there's a decent amount of repeated functionality here and in the AVS. Have you thought about moving the batching functionality up to a common folder, like you did the `constructCommitment` function? I was thinking you could just have one big function that you pass all the commits (identifier, time, and price you want to vote on), and it would construct and batch them, returning to you one (possibly large) data structure containing the batches, receipts, salts, etc. This may simplify the logic here and allow all the low-level batching and encryption stuff to happen in one place to isolate that complexity.

This also might allow us to isolate and write automated tests for that code without having to touch our CLI frontend.

Reply:
done